### PR TITLE
add some steganography tools

### PR DESCRIPTION
**Pull request by noraj1337**

### CODE REVIEW

**Comment by noraj1337** [comment]
commit 7d97560 : corrected the link where a **h** was missing

File: README.md [hunk 1]
```diff
@@ -200,16 +200,21 @@ Check solve section for steganography.
...
+## Steganography
 
 *Tools used for solving Steganography challenges*
 
 - [Convert](http://www.imagemagick.org/script/convert.php) - Convert images b/w formats and apply filters
+- [exif](http://manpages.ubuntu.com/manpages/trusty/man1/exif.1.html) - shows EXIF information in JPEG files
+- [exiftool](https://linux.die.net/man/1/exiftool) - Read and write meta information in files
+- [exiv2](http://www.exiv2.org/manpage.html) - Image metadata manipulation tool
 - [ImageMagick](http://www.imagemagick.org/script/index.php) - Tool for manipulating images
+- [outguess](https://www.freebsd.org/cgi/man.cgi?query=outguess+&apropos=0&sektion=0&manpath=FreeBSD+Ports+5.1-RELEASE&format=html) - universal steganographic tool
```
Comment (apsdehal) [inline README.md]: As a general rule for maintaining convention and standard, first letter of description should be captial.

File: README.md [hunk 2]
```diff
@@ -200,16 +200,21 @@ Check solve section for steganography.
...
 - [Request Bin](http://requestb.in/) - Lets you inspect http requests to a particular url
 
-## Stegano
+## Steganography
 
 *Tools used for solving Steganography challenges*
 
 - [Convert](http://www.imagemagick.org/script/convert.php) - Convert images b/w formats and apply filters
+- [exif](http://manpages.ubuntu.com/manpages/trusty/man1/exif.1.html) - shows EXIF information in JPEG files
+- [exiftool](https://linux.die.net/man/1/exiftool) - Read and write meta information in files
```
Comment (apsdehal) [inline README.md]: Same here.

Comment (apsdehal) [inline README.md]: Same goes for tool's name

File: README.md [hunk 3]
```diff
@@ -200,16 +200,21 @@ Check solve section for steganography.
...
+- [exiv2](http://www.exiv2.org/manpage.html) - Image metadata manipulation tool
 - [ImageMagick](http://www.imagemagick.org/script/index.php) - Tool for manipulating images
+- [outguess](https://www.freebsd.org/cgi/man.cgi?query=outguess+&apropos=0&sektion=0&manpath=FreeBSD+Ports+5.1-RELEASE&format=html) - universal steganographic tool
 - [pngtools](http://www.stillhq.com/pngtools/) - For various analysis related to PNGs
   - `apt-get install pngtools`
 - [SmartDeblur](https://github.com/Y-Vladimir/SmartDeblur) - Used to deblur and fix defocused images
 - [Steganabara](https://www.openhub.net/p/steganabara) -  Tool for stegano analysis written in Java
+- [stegbreak](https://linux.die.net/man/1/stegbreak) - launches brute-force dictionary attacks on JPG image
```
Comment (apsdehal) [inline README.md]: Same here

**Pull request merged**